docs/references/files/fossa-yml.md

```diff
@@ -150,6 +167,46 @@ The `labels` field allows you to add labels to projects so that you can classify
 
 Up to 5 labels are allowed to be associated with a project at a time. Labels will be applied in order up to the limit of 5 labels and then ignored.
 
+### `releaseGroup:`
+The releaseGroup field allows you to configure settings for the release group you are interacting with through the FOSSA API.
+
+>NOTE: releaseGroup configurations are only allowed for `fossa release-group create` and `fossa release-group add-project`. This is done so that release groups are not mistakenly deleted.
+
+>NOTE: release-group command line options will always take precedence over configurations set in `fossa.yml`. 
+
+#### `releaseGroup.title:`
+The title of the release group which can be seen in the FOSSA dashboard.
+
+#### `releaseGroup.release:`
+The release associated with the release group.
+
+#### `releaseGroup.releaseGroupProjects:`
+The projects associated with the release group's release. 
+
+>NOTE: At least one project must be specified upon creating a release group.
+
+>NOTE: `projectId` , `projectRevision`, and `projectBranch` must all be specified when providing a releaseGroupProject.
+
+#### `releaseGroup.releaseGroupProjects.projectId:`
+"The project ID defines a unique ID that the FOSSA API will use to reference this project. The project ID can be found in the UI on the project settings page listed as the `Project Locator` underneath the `Project Title` setting."
+
+>Note: When specifying `releaseGroup.releaseGroupProjects.projectId`, the full `Project Locator` value must be provided. For example, if the "Project Locator" value of `custom+1/foo` is provided in the FOSSA UI, use `custom+1/foo`. This implementation differs from `project.id`. 
```

Comment:
This is what the current documentation says about project.id:

```
If the 'Project Locator" value of `custom+1/foo` is provided in the FOSSA UI, use `foo`. 
```
https://github.com/fossas/fossa-cli/blob/b4b5d0b77cfcdac3fda7e53a0797adf678479e9e/docs/references/files/fossa-yml.md?plain=1#L108

I think this is because in commands that require a user to pass in a directory (analyze, container analyze, etc). We build up the project locator by inferring revision data from the directory and pass that to CORE. However, in the new commands (release-group , project), we don't have that and just need the user to pass the full project locator value to identify projects. 

I can see this being confusing to users because we're conflating how we use projectID with the addition of the `release-group` and `project` subcommands.  Maybe it might be better to have this be `releaseGroup.releaseGroupProjects.projectLocator`?

Reply:
Just saw your comment in the internal channel. I think the path going forward is this:

1. Change project.id -> project.locator to align with terminology in the UI
2. project.id -> Currently, project.id is just being used to update the name of the fossa project. So we can just update the docs and code path here:
https://github.com/fossas/fossa-cli/blame/b4b5d0b77cfcdac3fda7e53a0797adf678479e9e/src/App/Fossa/Config/Common.hs#L321

Reply:
After thinking about it more, I don't think we can get rid project.id. It's a specific portion of the project locator and it's used as a unique id to find projects within your organization. It's also used to target certain revisions when we run fossa analyze and container analyze. 

I kept the project.id implementation but just added more details in our docs to explain to users the difference between project.id and project.locator (what we will use for the release group commands).

Reply:
Docs are updated here:
https://github.com/fossas/fossa-cli/pull/1395/files